libs/RobotKit/include/RobotController.h

```diff
@@ -46,9 +46,9 @@ class RobotController : public interface::RobotController
 
 	explicit RobotController(interface::Timeout &timeout, interface::Battery &battery, SerialNumberKit &serialnumberkit,
 							 interface::FirmwareUpdate &firmware_update, interface::Motor &motor_left,
-							 interface::Motor &motor_right, interface::LED &ears, interface::LED &belt, LedKit &ledkit,
-							 interface::LCD &lcd, interface::VideoKit &videokit, BehaviorKit &behaviorkit,
-							 CommandKit &cmdkit, RFIDKit &rfidkit, ActivityKit &activitykit)
+							 interface::Motor &motor_right, interface::LED &ears, interface::LED &belt,
+							 interface::LedKit &ledkit, interface::LCD &lcd, interface::VideoKit &videokit,
+							 BehaviorKit &behaviorkit, CommandKit &cmdkit, RFIDKit &rfidkit, ActivityKit &activitykit)
```

Comment:
not related directly but we could also have an interface for behaviorkit, CommandKit, RFIDKit and ActivityKit to make testing easier